src/cond.js

```diff
@@ -18,18 +18,18 @@ var reduce = require('./reduce');
  * @since v0.6.0
  * @category Logic
  * @sig [[(*... -> Boolean),(*... -> *)]] -> (*... -> *)
- * @param {Array} pairs
+ * @param {Array} pairs A list of [predicat, transformer]
  * @return {Function}
  * @example
  *
  *      var fn = R.cond([
- *        [R.equals(0),   R.always('water freezes at 0°C')],
- *        [R.equals(100), R.always('water boils at 100°C')],
- *        [R.T,           temp => 'nothing special happens at ' + temp + '°C']
+ *        [R.lt(R.__, 0),    R.always("It's freezing out!")],
+ *        [R.gt(R.__, 40),  R.always("Yikes I'm getting sweaty!")],
+ *        [R.always(true), temp => "Nice! A balmy " + temp + " degrees!"]
  *      ]);
- *      fn(0); //=> 'water freezes at 0°C'
- *      fn(50); //=> 'nothing special happens at 50°C'
- *      fn(100); //=> 'water boils at 100°C'
+ *      fn(-10); //=> 'It's freezing out!'
+ *      fn(50); //=> 'Yikes I'm getting sweaty!'
+ *      fn(25); //=> 'Nice! A balmy 25 degrees!'
```

Comment:
i don't see the advantage of this one. The original is easier to parse without needing to know `__`